create-record/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "create-record",
+  "version": "1.0.2",
+  "description": "A serivce that takes in a JSON payload and puts it in an encrypted s3 bucket.",
+  "main": "index.js",
+  "dependencies": {
+    "aws-sdk": "^2.649.0",
+    "aws-sdk-mock": "^5.1.0",
+    "nodeunit": "^0.11.3",
```

Comment:
According to NPM nodeunit is deprecated and should not be used. 

https://www.npmjs.com/package/nodeunit

Reply:
ok. I'll check it out but jest was complaining about not have nodeunit installed